src/Home.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import "./Home.css";
+import { FeaturedItem } from "./FeaturedItem.js";
+import { SingleProduct } from "./SingleProduct.js";
+
+export class Home extends Component {
+  render() {
+    return (
+      <div className="home-page">
```

Comment:
What do you plan to show at the home page? Is it something other than the all-products view?